zerver/migrations/0043_realm_filter_validators.py

```diff
@@ -10,17 +7,4 @@ class Migration(migrations.Migration):
         ("zerver", "0042_attachment_file_name_length"),
     ]
 
-    operations = [
-        migrations.AlterField(
-            model_name="realmfilter",
-            name="pattern",
-            field=models.TextField(validators=[filter_pattern_validator]),
-        ),
-        migrations.AlterField(
-            model_name="realmfilter",
-            name="url_format_string",
-            field=models.TextField(
-                validators=[django.core.validators.URLValidator, filter_format_validator]
-            ),
-        ),
-    ]
+    operations = []
```

Comment:
Oh right, the empty list needs a type annotation like `operations: List[migrations.operations.base.Operation] = []`.

Reply:
Yes. We can probably remove this when we have django-stubs is integrated.